docusaurus.config.js

```diff
@@ -124,13 +124,13 @@ const darkCodeTheme = themes.dracula;
                     <img src="/img/main/cth-earth-logo.png" alt="CTH Earth Logo">
                     <div class="social-links">
                       <a href="https://chat.climatetechhandbook.com" target="_blank" rel="noreferrer noopener" aria-label="Slack">
-                        <img src="img/slack_icon.png" alt="Slack Icon" style="margin-right: 15px; width: 40px;"/>
+                        <img src="/img/main/slack_icon2.png" alt="Slack Icon" style="margin-right: 15px; width: 40px;"/>
```

Comment:
I grouped these images into the /img/main so that they would be in the same folder. I found that that path was working perfectly and the old path was creating an issue. So I just group the image together in the same folder.